DocsSnippets/01-UX/06-Customizing-Views/02-Changing-Appearance.swift

```diff
@@ -7,7 +7,7 @@ func snippets_ux_customizing_views_changing_appearance() {
     // > import UIKit
     // > import StreamChatUI
 
-    UIConfig.default.channelList.channelListItemView.defaultAppearance.addRule {
+    UIConfig.default.channelList.itemView.defaultAppearance.addRule {
         $0.backgroundColor = UIColor.yellow.withAlphaComponent(0.2)
         $0.titleLabel.textColor = .darkGray
     }
```

Comment:
Hmmm, this is not really what the snippets should look like. It's confusing. I'll fix it later when working on finalizing the documentation.